### PR TITLE
Borg Radio Repair

### LINKED ISSUE #1632: Borg radio does not get the channels it deserves
Calling recalculateChannels on the borg radio fixes the issue, but before that a borg will have none of the channels it is intended to have.
image
image
These are the only times the proc is called on a borg, which seems to imply that it does not get the proc called once it has channels assigned to it.

### PR DESCRIPTION
**Pull request by MciReloaded**
## About The Pull Request

Fixes #1632 
## Why It's Good For The Game

Allows borgs to speak in the proper channels without having to call the reset variable manually. 
## Changelog
:cl:
fix: #1632 Updates borg radio properly on creation of borg and changing of borg's type INSTEAD of whenever the hell it was doing it before.

/:cl:

### CODE REVIEW

**Comment by MciReloaded** [comment]
I know, I see the problem. Will re-upload with the properly compiled version.

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -79,9 +79,10 @@ var/global/list/robot_modules = list(
 	add_languages(R)
 	add_subsystems(R)
 	apply_status_flags(R)
+	R.radio.recalculateChannels()
 
-	if(R.radio)
-		R.radio.recalculateChannels()
+/*	if(R.radio)
+		R.radio.recalculateChannels()*/
```
Comment (EvilJackCarver) [inline code/modules/mob/living/silicon/robot/robot_modules.dm:85]: Untagged Eclipse edit

**Review by EvilJackCarver — CHANGES_REQUESTED** [review]
I'll leave it to Nestor to figure out whether the FILE_DIR definitions are necessary, since you're overwriting the definition over and over, but for now I've only got an untagged edit

**Comment by MciReloaded** [comment]
apologies on that, rookie mistake. I forgot to toggle that when I re-downloaded the repo.

**Review by EvilJackCarver — APPROVED** [review]

**Pull request merged**